src/Cache/Engine/ApcuEngine.php

```diff
@@ -68,15 +75,21 @@ public function write($key, $value)
      * Read a key from the cache
      *
      * @param string $key Identifier for the data
+     * @param mixed $default Default value
      * @return mixed The cached data, or false if the data doesn't exist,
      *   has expired, or if there was an error fetching it
      * @link https://secure.php.net/manual/en/function.apcu-fetch.php
      */
-    public function read($key)
+    public function get($key, $default = null)
     {
         $key = $this->_key($key);
 
-        return apcu_fetch($key);
+        $result = apcu_fetch($key);
+        if ($result === false) {
```

Comment:
I don't think this is correct. The success or failure of `apcu_fetch()` is determined by passing 2nd argument to the method.

http://php.net/manual/en/function.apcu-fetch.php